extension/source/QuillPage/Wallet/Contacts/ContactsWrapper.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { ContactsSummary } from './ContactsSummary';
+
+export interface IContacts {
+  name: string;
+}
+
+const contacts: IContacts[] = [
+  {
+    name: 'Andrew Hargreaves',
+  },
+  {
+    name: 'Angie Johnson',
+  },
+  {
+    name: 'Mom',
+  },
+];
+
+export const ContactsWrapper: React.FunctionComponent = () => {
+  const [selected, setSelected] = React.useState<number>(0);
```

Comment:
since this switching functionality is repeated multiple times, it can be refactored into a wrapper component but I didn't do it yet as I felt it won't refactor a lot of code and is sufficient at this stage.